lxc/rebuild.go

```diff
@@ -0,0 +1,186 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/lxc/lxd/client"
+	"github.com/lxc/lxd/lxc/config"
+	"github.com/lxc/lxd/lxc/utils"
+	"github.com/lxc/lxd/shared"
+	"github.com/lxc/lxd/shared/api"
+	cli "github.com/lxc/lxd/shared/cmd"
+	"github.com/lxc/lxd/shared/i18n"
+)
+
+// Rebuild.
+type cmdRebuild struct {
+	global    *cmdGlobal
+	flagEmpty bool
+}
+
+func (c *cmdRebuild) Command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = usage("rebuild", i18n.G("[[<remote>:]<image>...] [<remote>:]<instance>"))
+	cmd.Short = i18n.G("Rebuild instances")
+	cmd.Long = cli.FormatSection(i18n.G("Description"), i18n.G(
+		`Wipe the file system of an instance, but keep its configuration. By default, the original image is used to initialize the file system.`))
```

Comment:
Lets not use "filesystem" here as it wipes the root disk (which for VMs may contain multiple partitions with who knows what in).

Lets says:

> Wipe the instance root disk and re-initialize.

Reply:
Maybe change "By default, the original image is used to initialize the file system" to

> The original image is used to re-initialize the instance if a different image or --empty is not specified.